exodus/men_customization/templates/base/home.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base/base.html"%}
+{% block content %}
+{% load i18n %}
+{% load static %}
+<div class="row justify-content-center mb-5 mt-5">
+  <div class="col-xl-3 col-lg-3 col-md-3 col-sm-4 col-4 my-auto">
+    <img src="{% static "men_customization/logo-amal.png" %}" class="mx-auto d-block img-fluid" alt="amal logo">
+  </div>
+  <div class="col-xl-5 col-lg-5 col-md-8 col-sm-8 text-center my-auto">
+    <h1 class="main-title">AMAL</h1>
+    <h5><b>{% trans "Vos apps mobiles Android à la loupe." %}</b></h5>
+  </div>
```

Comment:
As it seems only the top part of the page is changing (the logo and the description), we should mutualise the template to not duplicate the whole code

Reply:
I won't factorize this part because we don't know if the UI customization is finished or not since we had no review from MEN.